test/config/newrelic.yml

```diff
@@ -41,6 +41,15 @@ test:
 
   error_collector:
     enabled: true
+    expected_messages:
+      StandardError:
```

Comment:
Could one of these classes be nested? I think it might be helpful to verify that YAML doesn't get confused in the future when it sees double colons, ex: `FakeController::FakeErrorClass`